lib/_debugger.js

```diff
@@ -829,13 +829,7 @@ function Interface(stdin, stdout, args) {
   // Run script automatically
   this.pause();
 
-  // XXX Need to figure out why we need this delay
-  setTimeout(function() {
-
-    self.run(function() {
-      self.resume();
-    });
-  }, 10);
+  setImmediate(() => { self.run(function() { self.resume(); }); });
```

Comment:
> `self.run(self.resume)`?

`test/parallel/test-debugger-util-regression.js` fails if `self.resume`/`this.resume` is not wrapped in a function. (I did not look to see why...)